frame/fast-unstake/src/lib.rs

```diff
@@ -15,6 +15,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! # Fast Unstake Pallet
+//!
+//! ## Overview
+//!
```

Comment:
Can you maybe mention that this pallet is covering an edge-case and it no good for mass migration of staking->pools?

Reply:
I'm kinda hesitant how to even phrase pools. This pallet knows nothing about the existence of pools 🙈